tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmRestApiIT.java

```diff
@@ -41,4 +100,52 @@ void platformChart_whenRestApiEnabled_thenCallsEndpoint() throws Exception {
             assertThat(body.jsonPath().getList("items")).isEmpty();
         }
     }
+
+    @Test
+    @Timeout(value = 5, unit = TimeUnit.MINUTES)
+    void platformChart_whenAuthEnabled_thenCallsEndpoint() throws Exception {
+        K8sUtil.createConfigMap(client, NAMESPACE, "ese-config-map.yml");
+        K8sUtil.createConfigMap(client, NAMESPACE, "ese-file-realm-config-map.yml");
+
+        HELM_CHART_CONTAINER.installPlatformChart(PLATFORM_RELEASE_NAME,
+                "-f",
+                "/files/rest-api-test-with-auth-values.yaml",
+                "--namespace",
+                NAMESPACE);
+
+        K8sUtil.waitForHiveMQPlatformStateRunning(client, NAMESPACE, PLATFORM_RELEASE_NAME);
+
+        // forward the port from the service
+        try (final var forwarded = K8sUtil.getPortForward(client,
+                NAMESPACE,
+                REST_API_SERVICE_NAME,
+                REST_API_SERVICE_PORT)) {
+            final var baseRestApiEndpoint = "http://localhost:" + forwarded.getLocalPort();
+
+            given().header("Authorization", createBasicAuthHeader("test-user", "test-password"))
```

Comment:
This looks very familiar with the ESE tests. I like it very much that you re-used the test idea/structure/code.

Reply:
Yeah, it's stolen from the ones we have there already :) - though I was kind of in doubt whether we really need to check that functionality here, but I guess there's no harm in double checking.